src/helpers.c

```diff
@@ -1028,7 +1028,10 @@ void print_regexp_error(int errcode, regex_t *regexp)
 
 bool is_url_allowed(char *url)
 {
-	if (strncmp(url, "http://", 7) == 0) {
+	char *url_lower;
+
+	url_lower = str_tolower(url);
```

Comment:
Really good catch.
But you're leaking memory here.
And you can use strncasecmp() instead of str_tolower() + strncmp()